### PR TITLE
Making the Laplacian element in the convection diffusion application use CONDUCTIVITY

### PR DESCRIPTION
**Pull request by jcotela**
As it says on the title. Current implementation always uses a conductivity coefficient of 1, this PR changes it to use the nodal CONDUCTIVITY variable. This is consistent with the convection-diffusion implementation. Also adding a test to verify the element.

### CODE REVIEW

File: applications/convection_diffusion_application/custom_elements/laplacian_element.cpp [hunk 1]
```diff
@@ -73,51 +75,54 @@ void LaplacianElement::CalculateLocalSystem(MatrixType& rLeftHandSideMatrix, Vec
...
     Matrix InvJ0(dim,dim);
     Vector temp(number_of_points);
 
     Vector heat_flux_local(number_of_points);
+    Vector nodal_conductivity(number_of_points);
     for(unsigned int node_element = 0; node_element<number_of_points; node_element++)
     {
-        heat_flux_local[node_element] = GetGeometry()[node_element].FastGetSolutionStepValue(HEAT_FLUX);
+        heat_flux_local[node_element] = r_geometry[node_element].FastGetSolutionStepValue(HEAT_FLUX);
+        nodal_conductivity[node_element] = r_geometry[node_element].FastGetSolutionStepValue(CONDUCTIVITY);
```
Comment (rubenzorrilla) [inline applications/convection_diffusion_application/custom_elements/laplacian_element.cpp]: Maybe if we do it as in the `eulerian_conv_diff` element we can reuse this same element for other "Laplacian" purposes. I mean to retrieve the conductivity variable from the convection diffusion settings.

What do you think?

Comment (jcotela) [reply applications/convection_diffusion_application/custom_elements/laplacian_element.cpp]: I thought about that, but the convection-diffusion settings have a lot of variables that would never be used here... We should consider using it, but maybe throwing errors if the unsupported variables (convection velocity, specific heat...) are defined.

Comment (rubenzorrilla) [reply applications/convection_diffusion_application/custom_elements/laplacian_element.cpp]: OK then. Let's leave it for another PR.

**Review by rubenzorrilla — DISMISSED** [review]

**Review by rubenzorrilla — APPROVED** [review]
Cool. Thanks for the implementation.

**Pull request merged**